docker/README.md

```diff
@@ -28,7 +28,7 @@ You can build all image via script in current dir.
 or just build only one image.
 
 ```shell
-./build.sh ams
+./build.sh amoro
 ```
 
 - NOTICE: The ams image and flink image required the project had been packaged. 
```

Comment:
There are some old instructions in the document. Can you change them together?
This should be 'amoro image and quickdemo image required ....'.

Reply:
> This should be 'amoro image and quickdemo image required ....'.

Please review it again ,thx

Reply:
The quickdemo image also required the project had been packaged.

Reply:
> The quickdemo image also required the project had been packaged.

done